packages/pangea-node-sdk/src/services/store.ts

```diff
@@ -0,0 +1,337 @@
+   * @operationId store_post_v1beta_delete
+   * @param {Store.DeleteRequest} request
+   * @returns {Promise} - A promise representing an async call to the delete endpoint.
+   * @example
+   * ```js
+   * const request = { id: "pos_3djfmzg2db4c6donarecbyv5begtj2bm" };
+   * const response = await client.delete(request);
+   * ```
+   */
+  delete(request: Store.DeleteRequest): Promise<PangeaResponse<Store.DeleteResult>> {
+    return this.post("v1beta/delete", request);
+  }
+
+  /**
+   * @summary Create a folder
+   * @description Create a folder, either by name or path and parent_id.
+   * @operationId store_post_v1beta_folder_create
+   * @param {Store.FolderCreateRequest} request
+   * @returns {Promise} - A promise representing an async call to the folder create endpoint.
+   * @example
+   * ```js
+   * const request = {
+   *   metadata: {
+   *     created_by: "jim",
+   *     priority: "medium",
+   *   },
+   *   parent_id: "pos_3djfmzg2db4c6donarecbyv5begtj2bm",
+   *   path: "/",
+   *   tags: ["irs_2023", "personal"],
+   * };
+   *
+   * const response = await client.folderCreate(request);
+   * ```
+   */
+  folderCreate(
+    request: Store.FolderCreateRequest
+  ): Promise<PangeaResponse<Store.FolderCreateResult>> {
+    return this.post("v1beta/folder/create", request);
+  }
+
+  /**
+   * @summary Get an object
+   * @description Get object. If both ID and Path are supplied, the call will fail if the target object doesn't match both properties.
+   * @operationId store_post_v1beta_get
+   * @param {Store.GetRequest} request
+   * @returns {Promise} - A promise representing an async call to the get item endpoint.
+   * @example
+   * ```js
+   * const request = {
+   *   id: "pos_3djfmzg2db4c6donarecbyv5begtj2bm",
+   *   path: "/",
+   * };
+   *
+   * const response = await client.getItem(request);
+   * ```
+   */
+  getItem(request: Store.GetRequest): Promise<PangeaResponse<Store.GetResult>> {
+    return this.post("v1beta/get", request);
+  }
+
+  /**
+   * @summary Get archive
+   * @description Get an archive file of multiple objects.
+   * @operationId store_post_v1beta_get_archive
+   * @param {Store.GetArchiveRequest} request
+   * @returns {Promise} - A promise representing an async call to the get archive endpoint.
+   * @example
+   * ```js
+   * const request = { ids: ["pos_3djfmzg2db4c6donarecbyv5begtj2bm"] };
+   * const response = await client.getArchive(request);
+   * ```
+   */
+  getArchive(request: Store.GetArchiveRequest): Promise<PangeaResponse<Store.GetArchiveResult>> {
+    return this.post("v1beta/get_archive", request);
+  }
+
+  /**
+   * @summary List
+   * @description List or filter/search records.
+   * @operationId store_post_v1beta_list
+   * @param {Store.ListRequest} request
+   * @returns {Promise} - A promise representing an async call to the list endpoint.
+   * @example
+   * ```js
+   * const request = {};
```

Comment:
Kinda feel like putting an empty object here is a bit superfluous. If we're not gonna demonstrate any of the cool list request options then consider just not having this here at all.

Going further, if an empty object is okay as input here, we should probably have the function's parameter be optional or default to the empty object.

Reply:
Yes, this should be an optional param. I'll fix it. The only filter already defined I think it's `folder`, we can add it to the example.